src/main/java/com/liferay/faces/test/selenium/IntegrationTesterBase.java

```diff
@@ -127,6 +127,6 @@ protected final void signIn(Browser browser, String signInURL, String loginXpath
 		WebElement loginElement = browser.findElementByXpath(loginXpath);
 		browser.click(signInButtonXpath);
 		browser.waitUntil(ExpectedConditions.stalenessOf(loginElement));
-		browser.waitForElementVisible("//body");
+		browser.waitForElementDisplayed("//body");
```

Comment:
@stiemannkj1 I wasn't suggesting that "visible" would be good here instead of "displayed" -- could "elementPresent" be used instead?